tests/models/test_modeling_common.py

```diff
@@ -1519,3 +1523,188 @@ def test_push_to_hub_library_name(self):
 
         # Reset repo
         delete_repo(self.repo_id, token=TOKEN)
+
+
+class TestLoraHotSwappingForModel(unittest.TestCase):
```

Comment:
We could mark the entire class here instead of marking the individual methods.

Reply:
Good point, done.